cmd/src/cli/scaffold/rust.rs

```diff
@@ -36,8 +37,11 @@ fn generate_cargo_toml(name: &str, contents: &str) -> DefaultResult<String> {
     let edition = maybe_package
         .and_then(|p| p.get("edition"))
         .unwrap_or(&edition_default);
+    let branch = maybe_package
```

Comment:
@zippy why would a branch be set in the package? does package mean cargo.toml?

Reply:
My intent is that you can specify which branch of holochain you are are building the package against in your cargo toml.  The `maybe_package` was already there so I just used it, I think it actually should by `maybe_cargo`

Reply:
Actually it the "package" section in the cargo file, and there's no branch in there.  So that's bogus.  I'll fix that.